src/gateway.ts

```diff
@@ -70,6 +70,17 @@ export class GatewayClient extends EventEmitter {
 		);
 	}
 
+	public async sendTypingPacket(channelID: string) {
```

Comment:
I appreciate the reasoning for this method, but that leads to client inconsistencies - e.g. why is there a method for typing packets but no others?

Since all this method does is call .send and nothing too complicated, can you remove this just for consistency?

Reply:
Yeah makes sense. So we're gonna call `apiClient.gateway.send<ClientTypingPacket>()` directly in the frontend? no other processing that needs to be done?

Reply:
Yep!